certified-connectors/tegolySIGN/apiDefinition.swagger.json

```diff
@@ -0,0 +1,164 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "tegolySIGN",
+    "description": "Automate your signatures with the tegolySIGN connector. You can transform manual processes into fully digital ones. Integrate tegolySIGN functions into your favourite applications and create fully automated signature workflows.",
+    "version": "1.0",
+    "contact": {
+      "name": "tegolySIGN Support",
+      "url": "https://tegoly.com/support",
+      "email": "support@tegoly.com"
+    }
+  },
+  "host": "tegolysignbackend.azurewebsites.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/webhook/end-trigger": {
+      "x-ms-notification-content": {
+        "schema": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string",
+              "description": "Name of the PDF document"
+            },
+            "document": {
+              "type": "string",
+              "format": "byte",
+              "description": "The signed PDF document"
+            },
+            "certificate": {
+              "type": "string",
+              "format": "byte",
+              "description": "The certificate for the signed doucment"
+            }
+          }
+        },
+        "description": "Returns the signed document, name and certificate"
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
```

Comment:
@JanGosda The 'default' response should not have schema definition. Schemas should be defined on expected responses only.